standard/clause_6_informative_text.adoc

```diff
@@ -481,6 +481,17 @@ To identify a predicate as a variable-to-variable reference predicate, that pred
   <{predicate}> rdfs:range bald:Resource .
 ----
 
+Alternatively, it is permissable to provide an rdfs:range statement targeting another resource and for that resource to declare that it is an rdfs:subClassOf bald:Resource:
+
+----
+ <{attribute}> rdfs:range <{AClass}> .
+
+ <{AClass}> rdfs:subClassOf <https://www.opengis.net/def/binary-array-ld/Resource> .
+----
+
+This standard does not support nested derivation, an implementation is not required to search the inheritance tree of a defined Class to see if any of the classes it inherits from define themselves as rdfs:subClassOf bald:Resource.
+
+The definition of the owl:objectProperty bald:references uses this rdfs:range, rdfs:subClassOf definition within the bald ontology: https://www.opengis.net/def/binary-array-ld
```

Comment:
Should we provide the definition of the bald:references objectProperty inline here to be explicit about what it is?

Reply:
added